docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Introducing the TestCafe Jenkins plugin"
+permalink: /media/team-blog/:title.html
+isTeamBlog: true
+author: Pavel Redrukhin, Eugene Titerman
+---
+# Introducing the TestCafe Jenkins plugin
+
+TestCafe offers a number of debugging features, including the capability to automatically capture videos and screenshots during the testing process. You can record all your tests or only those that failed, generate multiple video files or stitch them together, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).
+
+But automating this process was not always easy. For a long time, Jenkins users had to manually match completed tests to their artifacts. The recent introduction of the [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) changes that. Links to screenshots and videos taken during the test now automatically appear on the Jenkins test results page. You don't even need to keep your testing server running — all the necessary files are stored right inside the Jenkins build folder.
+
+<!--more-->
+
+The test results page should look like this with our plugin enabled:
+
+![The updated test results page](/testcafe/images/jenkins-plugin/test-results-page.png)
+
+## Test case: the disappearing button
+
+Let's say, the 'Add to Cart' button keeps disappearing from the mobile version of your website.
+
+![The 'Add To Cart' button is absent from the mobile version of the website](/testcafe/images/jenkins-plugin/button-example.png)
+
+There are several ways TestCafe can help you detect this issue. You can:
+
+* Run the test through a [3rd party cloud testing service](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#browsers-in-cloud-testing-services)
+* [Resize the browser window](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html#resize-window) during the test, or
+* Take advantage of [Chromium's built-in emulation mode](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#use-chromium-device-emulation)
+
+For the sake of this tutorial, let's pick the third option, as it offers greater stability and speed.
```

Comment:
"For the sake of" should be followed by something else - a word that describes how this choice affects the result. "For the sake of simplicity", "for the sake of completeness". "For the sake of this tutorial" doesn't really mean anything.

You also seem to imply that this third option is generally better than the other two... so here's my version:

```suggestion
The third option produces most stable and quick tests. We describe this option going forward and recommend that you use it if possible. 
```

Reply:
Suggestion:
```markdown
The third option offers greater speed and stability. We describe it going forward and recommend that you use it if possible.
```
Rationale: Potentially misleading verb choice (emulation cannot produce tests)